packages/interface/src/store/test/selectors.js

```diff
@@ -1,107 +0,0 @@
-/**
- * Internal dependencies
- */
-import { isFeatureActive } from '../selectors';
-
-describe( 'selectors', () => {
-	describe( 'isFeatureActive', () => {
```

Comment:
I've deleted this as the new registry selector is tested extensively in the actions.js test file.